### PR TITLE
Refine stdlib commands module (add mkarg(), mk2arg(), correct get*())

### PR DESCRIPTION
**Pull request by moreati**
I wasn't sure either way about adding these. I've done so on the basis of "If in doubt, err on the side of including more objects", and because there is code in the wild that uses them. Reject this PR at
will.

For the record they are

- Explicitly documented as [undocumented](https://docs.python.org/2/library/commands.html)
- not in `commands.__all__`
- present in all Python 2.x releases
- unchanged since March 1998.

The module was deprecated in Python 2.6 and removed in 3.0.

### CODE REVIEW

File: stdlib/2/commands.pyi [hunk 1]
```diff
@@ -3,3 +3,6 @@ from typing import Tuple
 def getstatus(file: str) -> str: ...
 def getoutput(cmd: str) -> str: ...
 def getstatusoutput(cmd: str) -> Tuple[int, str]: ...
+
+def mk2arg(head: str, x: str) -> str: ...
+def mkarg(x: str) -> str: ...
```
Comment (JelleZijlstra) [inline stdlib/2/commands.pyi]: This should be AnyStr -> AnyStr I think.

File: stdlib/2/commands.pyi [hunk 2]
```diff
@@ -3,3 +3,6 @@ from typing import Tuple
 def getstatus(file: str) -> str: ...
 def getoutput(cmd: str) -> str: ...
 def getstatusoutput(cmd: str) -> Tuple[int, str]: ...
+
+def mk2arg(head: str, x: str) -> str: ...
```
Comment (JelleZijlstra) [inline stdlib/2/commands.pyi]: Strictly speaking this should have a similar signature to `os.path.join` (https://github.com/python/typeshed/blob/master/stdlib/2/os/path.pyi#L64), but I think it's simpler and almost correct to make all the arguments and return values be Text.

File: stdlib/2/commands.pyi (hunk 1 above)
Comment (moreati) [reply stdlib/2/commands.pyi]: I presume that whatever the answer is for `mkarg()`, should also be applied to the 3 get* functions in this module.

File: stdlib/2/commands.pyi (hunk 2 above)
Comment (moreati) [reply stdlib/2/commands.pyi]: Thank you, I'll look at this in more detail

File: stdlib/2/commands.pyi (hunk 1 above)
Comment (JelleZijlstra) [reply stdlib/2/commands.pyi]: No, for them the argument type doesn't affect the return type. I think their argument types should be Text (which means in Python 2 they accept both `str` and `unicode`) and their return types can stay as they are.

File: stdlib/2/commands.pyi (hunk 2 above)
Comment (JelleZijlstra) [reply stdlib/2/commands.pyi]: I made a small adjustment to the overloads you added—merging the last two into one still works correctly. Text (=unicode) in Python 2 automatically includes bytes (=str).

**Pull request merged**